cscs-checks/tools/profiling_and_debugging/notool.py

```diff
@@ -47,94 +47,64 @@ def restrict_f90_concurrency(self):
 
-    @run_before('run')
-    def set_prerun_cmds(self):
-        if self.current_system.name in {'dom', 'daint', 'eiger', 'pilatus'}:
-            # get general info about the environment:
-            self.prerun_cmds += ['module list']
-            self.prerun_cmds += [
-                # cray/aocc compilers version are needed but others won't hurt:
-                f'echo CRAY_CC_VERSION=$CRAY_CC_VERSION',
-                f'echo CRAY_AOCC_VERSION=$CRAY_AOCC_VERSION',
-                f'echo GNU_VERSION=$GNU_VERSION',
-                f'echo PGI_VERSION=$PGI_VERSION',
-                f'echo INTEL_VERSION=$INTEL_VERSION',
-                f'echo INTEL_COMPILER_TYPE=$INTEL_COMPILER_TYPE',
-            ]
-
-    @run_before('run')
-    def set_postrun_cmds(self):
-        readme_str = (
-            rf'More debug and performance tools ReFrame checks are '
-            rf'available at {self.url}'
-        )
-        self.postrun_cmds += [f'echo "{readme_str}"']
-
-
-
     @run_before('compile')
     def prgEnv_nvidia_workaround(self):
-        # {{{ CRAY_MPICH_VERSION
-        # cdt/20.08 cray-mpich/7.7.15 cray, crayclang, gnu, intel, pgi
-        # cdt/21.02 cray-mpich/7.7.16 cray, crayclang, gnu, intel, pgi
-        # cdt/21.05 cray-mpich/7.7.17 crayclang, gnu, intel, pgi, *nvidia*
-        #
-        # cpe/21.04 cray-mpich/8.1.4 AOCC, CRAY, CRAYCLANG, GNU, INTEL, NVIDIA
-        # cpe/21.05 cray-mpich/8.1.5 AOCC, CRAY, CRAYCLANG, GNU, INTEL, NVIDIA
-        # cpe/21.06 cray-mpich/8.1.6 AOCC, CRAY, CRAYCLANG, GNU, INTEL, NVIDIA
-        # cpe/21.08 cray-mpich/8.1.8 AOCC, CRAY, CRAYCLANG, GNU, INTEL, NVIDIA
-        # }}}
         envname = self.current_environ.name
         sysname = self.current_system.name
-        self.cppflags = ''
-        if (sysname in ['dom', 'daint'] and envname == 'PrgEnv-nvidia'):
-            mpi_version = int(os.getenv('CRAY_MPICH_VERSION').replace('.', ''))
-            self.skip_if(
-                mpi_version <= 7716,
-                (f'PrgEnv-nvidia not supported with cray-mpich<=7.7.16 '
-                 f'(CRAY_MPICH_VERSION={mpi_version})'))
-            # NOTE: occasionally, the wrapper fails to find the mpich dir
-            mpich_pkg_config_path = '$CRAY_MPICH_PREFIX/lib/pkgconfig'
-            self.variables = {
-                'PKG_CONFIG_PATH': f'$PKG_CONFIG_PATH:{mpich_pkg_config_path}'
-            }
-            self.cppflags = ('`pkg-config --cflags mpich` '
-                             '`pkg-config --libs mpich`')
-        elif sysname in ['pilatus', 'eiger']:
-            self.skip_if(self.current_environ.name == 'PrgEnv-nvidia', '')
+        if sysname in ['pilatus', 'eiger', 'dom', 'daint']:
+            # FIXME: Fatal error in PMPI_Init_thread
+            self.skip_if(envname == 'PrgEnv-nvidia', '')
 
     @run_before('compile')
     def set_flags(self):
         # FIXME: workaround for C4KCUST-308
-        self.modules += ['cray-mpich']
+        # self.modules += ['cray-mpich']
```

Comment:
Remove these two completely, if they are not actually needed anymore.

Reply:
https://github.com/eth-cscs/reframe/pull/2298/commits/94516b25e5c7b6d8dbf933c18df2680db76c0608